src/pages/downloads.tsx

```diff
@@ -0,0 +1,233 @@
+import {
+  Code,
+  Link,
+  ListItem,
+  Stack,
+  Text,
+  UnorderedList,
+} from '@chakra-ui/react';
+import type { NextPage } from 'next';
+import { useState } from 'react'
+
+import {
+  DownloadsHero,
+  DownloadsSection,
+  DownloadsTable,
+} from '../components/UI/downloads';
+import { DataTable } from '../components/UI/DataTable';
+
+import {
+  DEFAULT_BUILD_AMOUNT_TO_SHOW,
+  GETH_REPO_URL
+} from '../constants'
+
+import { testDownloadData } from '../data/test/download-testdata'
+import { pgpBuildTestData } from '../data/test/pgpbuild-testdata';
+import { pgpDeveloperTestData } from '../data/test/pgpdeveloper-testdata';
+
+const DownloadsPage: NextPage = ({}) => {
+  const [amountStableReleases, updateAmountStables] = useState(DEFAULT_BUILD_AMOUNT_TO_SHOW)
+  const [amountDevelopBuilds, updateAmountDevelopBuilds] = useState(DEFAULT_BUILD_AMOUNT_TO_SHOW)
+
+  return (
+    <>
+     {/* TODO: add PageMetadata */}
+     
+     <main>
+      <Stack spacing={4}>
+        {/* TODO: replace hardcoded strings with build information */}
+        <DownloadsHero
+          currentBuildName={'Sentry Omega'}
+          currentBuildVersion={'v1.10.23'}
+          linuxBuildURL={'https://gethstore.blob.core.windows.net/builds/geth-linux-amd64-1.10.25-69568c55.tar.gz'}
+          macOSBuildURL={'https://gethstore.blob.core.windows.net/builds/geth-darwin-amd64-1.10.25-69568c55.tar.gz'}
+          releaseNotesURL={''}
+          sourceCodeURL={'https://github.com/ethereum/go-ethereum/archive/v1.10.25.tar.gz'}
+          windowsBuildURL={'https://gethstore.blob.core.windows.net/builds/geth-windows-amd64-1.10.25-69568c55.exe'}
+        />
+
```

Comment:
`DownloadsSection` (stable, develop) could be mapped to avoid repetition

Reply:
I think this might make sense to look at refactoring when we are doing logic to see if it makes sense. If you are ok with that.